datafusion/common/src/scalar.rs

```diff
@@ -959,6 +959,9 @@ impl ScalarValue {
             ScalarValue::Decimal256(Some(v), precision, scale) => Ok(
                 ScalarValue::Decimal256(Some(v.neg_wrapping()), *precision, *scale),
             ),
+            ScalarValue::TimestampSecond(Some(v), tz) => {
```

Comment:
I wonder if we should also support `TimestampMillisecond`, `TimestampMicrosecond` and `TimestampNanosecond` 🤔